internal/testutils/bubblewrap.go

```diff
@@ -48,19 +48,27 @@ func SkipIfCannotRunBubbleWrap(t *testing.T) {
 	t.Skip("Skipping test: requires root privileges or unprivileged user namespaces")
 }
 
+// RunInBubbleWrapWithEnv runs the passed commands in bubble wrap sandbox with env variables.
```

Comment:
In commit message of b419d537cb1c3b71350aa996337d214a544ae0ed:

```diff
If another process is locking the shadow password database authd will
-hang until the calling process will not be done with it.
+hang until the calling process will be done with it.
```

Reply:
Ah, italianism! :)